### DISCUSSION TITLE
unable to change login email

### DISCUSSION THREAD

**Discussion by serendrewpity** [emailjs-com/help#112] [General]
Current email address: oevzdstmdafmlndlih@cazlq.com
Desired Email address: buymyregalhome@gmail.com

I tried to change my account email address under 'account' > 'Notifications', but I cannot log in with my new, desired email address. I am still required to log in with the old address. How do I change this?

**Comment by xr0master** [comment]
You can contact support team from the dashboard. They will change it for you.
The Notifications email is for account notifications.

**Comment by mcarter-astronautdev** [comment]
@xr0master I have the same issue, but it's not possible to contact support from the dashboard with a free account.  Do I have to upgrade my account to a paid plan in order to change the login email?

**Reply by xr0master** [reply]
We’ve disabled the contact form for Free plans, as it was heavily abused and generated too many requests with very basic questions. Currently, there is no way to change the login email address on a free account.

The available option is to create a new account with the correct email, migrate your data there, and then delete the old account once you’re done. Or, upgrade to a paid plan (even for one month) and then contact our support team to change the email for you.
